{"name": "用户注册登录修改--预期失败", "status": "failed", "statusDetails": {"message": "AssertionError: 接口返回码是 【 1003 】, 返回信息：用户名不存在！！！ \nassert False is True\n +  where False = <core.result_base.ResultBase object at 0x1076b3a90>.success", "trace": "self = <testcases.scenario_test.test_02_register_login_update.TestRegLogUpdate object at 0x1076b5ac0>\ntestcase_data = {'except_code': 4005, 'except_msg': '修改的用户ID不存在', 'except_result': False, 'login': {'admin_pwd': '123456', 'admin_user': 'wintest'}, ...}\n\n    @allure.story(\"用例--注册/登录/修改--预期失败\")\n    @allure.description(\"该用例是针对 注册-登录-修改 场景的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"用户注册登录修改--预期失败\")\n    @pytest.mark.multiple\n    @pytest.mark.usefixtures(\"delete_register_user\")\n    def test_user_register_login_update_fail(self, testcase_data):\n        username = testcase_data[\"register\"][\"username\"]\n        password = testcase_data[\"register\"][\"password\"]\n        telephone = testcase_data[\"register\"][\"telephone\"]\n        sex = testcase_data[\"register\"][\"sex\"]\n        address = testcase_data[\"register\"][\"address\"]\n        admin_user = testcase_data[\"login\"][\"admin_user\"]\n        admin_pwd = testcase_data[\"login\"][\"admin_pwd\"]\n        new_password = testcase_data[\"update\"][\"new_password\"]\n        new_telephone = testcase_data[\"update\"][\"new_telephone\"]\n        new_sex = testcase_data[\"update\"][\"new_sex\"]\n        new_address = testcase_data[\"update\"][\"new_address\"]\n        except_result = testcase_data[\"except_result\"]\n        except_code = testcase_data[\"except_code\"]\n        except_msg = testcase_data[\"except_msg\"]\n        logger.info(\"*************** 开始执行用例 ***************\")\n        result = register_user(username, password, telephone, sex, address)\n        step_1(username, password, telephone, sex, address)\n        assert result.success is True, result.error\n        result = login_user(admin_user, admin_pwd)\n        step_2(admin_user)\n>       assert result.success is True, result.error\nE       AssertionError: 接口返回码是 【 1003 】, 返回信息：用户名不存在！！！ \nE       assert False is True\nE        +  where False = <core.result_base.ResultBase object at 0x1076b3a90>.success\n\ntestcases/scenario_test/test_02_register_login_update.py:102: AssertionError"}, "description": "该用例是针对 注册-登录-修改 场景的测试", "steps": [{"name": "步骤1 ==>> 注册用户", "status": "passed", "parameters": [{"name": "username", "value": "'测试test'"}, {"name": "password", "value": "'123456'"}, {"name": "telephone", "value": "'13599999944'"}, {"name": "sex", "value": "'1'"}, {"name": "address", "value": "'深圳市宝安区'"}], "start": 1744986709644, "stop": 1744986709644}, {"name": "步骤2 ==>> 登录用户", "status": "passed", "parameters": [{"name": "username", "value": "'wintest'"}], "start": 1744986709694, "stop": 1744986709694}], "attachments": [{"name": "log", "source": "ba029bb8-273a-4f83-b596-28ebfde85d48-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "0eda02a3-33e7-49d6-a614-001a140c38fa-attachment.txt", "type": "text/plain"}], "start": 1744986709568, "stop": 1744986709695, "uuid": "66c01ff2-62c2-4d38-9747-500568979cc9", "historyId": "5e88704a520d70faf3165b42517fe6c2", "testCaseId": "b56ec6e5f1e8b3339a5dd14be9d281c0", "fullName": "testcases.scenario_test.test_02_register_login_update.TestRegLogUpdate#test_user_register_login_update_fail", "labels": [{"name": "story", "value": "用例--注册/登录/修改--预期失败"}, {"name": "epic", "value": "针对业务场景的测试"}, {"name": "severity", "value": "blocker"}, {"name": "feature", "value": "场景：用户注册-用户登录-修改用户"}, {"name": "tag", "value": "multiple"}, {"name": "parentSuite", "value": "testcases.scenario_test"}, {"name": "suite", "value": "test_02_register_login_update"}, {"name": "subSuite", "value": "TestRegLogUpdate"}, {"name": "host", "value": "192.168.10.5"}, {"name": "thread", "value": "42260-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.scenario_test.test_02_register_login_update"}], "links": [{"type": "test_case", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应BUG的链接地址"}]}